### PR TITLE
telecabine -> telecabin

### PR DESCRIPTION
**Pull request by ue71603**

### CODE REVIEW

**Comment by skinkie** [comment]
Breaking change, but since this is a typo in an enumeration, it must be fixed asap.

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd [hunk 1]
```diff
@@ -414,7 +414,7 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="positionAtBusStop"/>
 			<xsd:enumeration value="boatGangway"/>
 			<xsd:enumeration value="ferryGangway"/>
-			<xsd:enumeration value="telecabineplatform"/>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd:417]: Is there any reason why the casing is not the same between the two instances? If we change this, can't we directly align it?

**Review by Aurige — APPROVED** [review]
not great for compatibility, but OK (shouldn't we keep a deprecated buggy value ?)

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd:417]: done

**Review by skinkie — APPROVED** [review]

**Pull request merged**